lib/util/index.js

```diff
@@ -481,12 +481,10 @@ function checkIfSdpSemanticsIsSupported() {
  * @returns {void}
  */
 function hidePrivateProperties(object) {
-  Object.getOwnPropertyNames(object).forEach(name => {
-    if (name.startsWith('_')) {
-      const descriptor = Object.getOwnPropertyDescriptor(object, name);
-      descriptor.enumerable = false;
-      Object.defineProperty(object, name, descriptor);
-    }
+  Object.getOwnPropertyNames(object).filter(name => {
+    return name.startsWith('_');
+  }).forEach(name => {
+    hideProperty(object, name);
   });
```

Comment:
The number of properties here is small, so it doesn't really matter, but `filter` followed by `forEach` is going to traverse twice, whereas keeping the `if`-statement inside traverses once. If you do keep the `filter`, I'd at least recommend making it as terse as possible, i.e.

```js
Object.getOwnPropertyNames(object)
  .filter(name => name.startsWith('_'))
  .forEach(name => hideProperty(object, name));
```